docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,17 @@
+# .Net
```

Comment:
Let's use `Cloudstate .Net` like the other projects.

Reply:
Ok